src/Examine.AzureDirectory/AzureHelper.cs

```diff
@@ -0,0 +1,99 @@
+﻿using System.Diagnostics;
+using System.IO;
+using System.IO.Compression;
+using Azure.Storage.Blobs;
+using Examine.LuceneEngine.Directories;
+
+namespace Examine.AzureDirectory
+{
+    public static class AzureHelper
```

Comment:
I don't think this refactoring is appropriate. Static helper classes are not possible to mock in tests.